[2014-04-23 01:38:51,492][WARN ][monitor.jvm              ] [Photon] [gc][ParNew][65610][10] duration [52.9s], collections [1]/[1m], total [52.9s]/[58.5s], memory [129.3mb]->[27.3mb]/[1011.2mb], all_pools {[Code Cache] [1.8mb]->[1.8mb]/[48mb]}{[Par Eden Space] [102.5mb]->[3.1mb]/[102.5mb]}{[Par Survivor Space] [5.6mb]->[1.7mb]/[12.7mb]}{[CMS Old Gen] [21.2mb]->[22.4mb]/[896mb]}{[CMS Perm Gen] [19.7mb]->[19.7mb]/[128mb]}
[2014-04-23 01:41:02,155][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 01:42:05,563][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 01:43:06,558][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 01:44:07,884][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 01:45:08,660][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 01:46:08,798][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 01:47:08,945][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 01:48:09,073][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 01:49:09,177][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 01:50:09,266][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 01:51:09,406][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 01:52:09,680][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 01:53:10,935][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 01:54:11,147][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 01:55:11,482][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 01:56:11,637][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 01:57:11,764][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 01:58:11,875][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 01:59:11,970][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:00:12,073][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:01:12,146][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:02:12,274][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:03:12,382][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:04:15,111][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:05:15,480][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:05:40,809][INFO ][cluster.metadata         ] [Photon] [users] deleting index
[2014-04-23 02:05:46,178][INFO ][cluster.metadata         ] [Photon] [users] creating index, cause [api], shards [5]/[1], mappings [user]
[2014-04-23 02:05:57,670][INFO ][cluster.metadata         ] [Photon] [users] update_mapping [user] (dynamic)
[2014-04-23 02:06:15,580][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:07:15,639][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:08:15,673][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:09:15,709][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:10:15,738][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:10:50,169][INFO ][cluster.metadata         ] [Photon] [photos] deleting index
[2014-04-23 02:10:51,668][INFO ][cluster.metadata         ] [Photon] [photos] creating index, cause [api], shards [5]/[1], mappings [photo]
[2014-04-23 02:11:15,764][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:11:53,322][WARN ][monitor.jvm              ] [Photon] [gc][ParNew][67534][12] duration [1.6s], collections [1]/[1.7s], total [1.6s]/[1m], memory [140.6mb]->[46.5mb]/[1011.2mb], all_pools {[Code Cache] [2.8mb]->[2.8mb]/[48mb]}{[Par Eden Space] [101mb]->[1.3mb]/[102.5mb]}{[Par Survivor Space] [12.7mb]->[10.7mb]/[12.7mb]}{[CMS Old Gen] [26.8mb]->[34.3mb]/[896mb]}{[CMS Perm Gen] [21.1mb]->[21.1mb]/[128mb]}
[2014-04-23 02:11:54,153][INFO ][cluster.metadata         ] [Photon] [photos] update_mapping [photo] (dynamic)
[2014-04-23 02:12:15,794][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:13:19,437][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:14:19,471][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:15:19,548][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:16:19,612][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:17:19,648][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:18:19,734][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:19:19,820][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:20:03,234][INFO ][cluster.metadata         ] [Photon] [cities] deleting index
[2014-04-23 02:20:03,732][INFO ][cluster.metadata         ] [Photon] [cities] creating index, cause [api], shards [5]/[1], mappings [city]
[2014-04-23 02:20:19,842][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:20:26,687][INFO ][cluster.metadata         ] [Photon] [cities] update_mapping [city] (dynamic)
[2014-04-23 02:21:19,886][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:22:19,915][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:23:19,959][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:24:20,008][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:25:20,045][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:26:20,087][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:27:20,140][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:28:20,169][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:29:20,208][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:30:20,244][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:31:20,279][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:32:20,312][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:33:20,344][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:34:20,373][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:35:20,949][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:36:21,019][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:37:23,569][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:38:23,752][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:39:23,898][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:39:38,930][INFO ][cluster.metadata         ] [Photon] [pois] deleting index
[2014-04-23 02:39:39,680][INFO ][cluster.metadata         ] [Photon] [pois] creating index, cause [api], shards [5]/[1], mappings [poi]
[2014-04-23 02:40:24,000][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:40:29,702][INFO ][cluster.metadata         ] [Photon] [pois] update_mapping [poi] (dynamic)
[2014-04-23 02:40:31,372][INFO ][monitor.jvm              ] [Photon] [gc][ParNew][69237][24] duration [920ms], collections [1]/[1.4s], total [920ms]/[1m], memory [111.6mb]->[73mb]/[1011.2mb], all_pools {[Code Cache] [3.9mb]->[3.9mb]/[48mb]}{[Par Eden Space] [46mb]->[298.6kb]/[102.5mb]}{[Par Survivor Space] [12.7mb]->[10.6mb]/[12.7mb]}{[CMS Old Gen] [52.8mb]->[62.1mb]/[896mb]}{[CMS Perm Gen] [23.1mb]->[23.1mb]/[128mb]}
[2014-04-23 02:41:24,070][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:42:24,263][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:43:24,290][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:44:08,023][INFO ][cluster.metadata         ] [Photon] [pois] update_mapping [poi] (dynamic)
[2014-04-23 02:44:24,313][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:45:24,352][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:46:24,390][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:47:24,418][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:48:24,547][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:49:24,730][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:50:24,767][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:51:24,919][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:52:24,995][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:53:25,027][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2014-04-23 02:54:25,103][WARN ][cluster.service          ] [Photon] failed to reconnect to node [Photon][i8eZj5JMR_SfIzWbFAwAqQ][inet[/192.168.1.105:9300]]
org.elasticsearch.transport.ConnectTransportException: [Photon][inet[/192.168.1.105:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:671)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:610)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:580)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:401)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: java.net.ConnectException: Connection refused: /192.168.1.105:9300
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:708)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.connect(NioClientBoss.java:150)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.processSelectedKeys(NioClientBoss.java:105)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.process(NioClientBoss.java:79)
	at org.elasticsearch.common.netty.channel.socket.nio.AbstractNioSelector.run(AbstractNioSelector.java:312)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientBoss.run(NioClientBoss.java:42)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:108)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    [2014-04-23 02:58:57,284][INFO ][node                     ] [MacPherran, Mary "Skeeter"] {0.90.0}[2323]: initializing ...
[2014-04-23 02:58:57,476][INFO ][plugins                  ] [MacPherran, Mary "Skeeter"] loaded [analysis-kuromoji], sites []
[2014-04-23 02:59:13,615][INFO ][node                     ] [MacPherran, Mary "Skeeter"] {0.90.0}[2323]: initialized
[2014-04-23 02:59:13,616][INFO ][node                     ] [MacPherran, Mary "Skeeter"] {0.90.0}[2323]: starting ...
[2014-04-23 02:59:14,992][INFO ][transport                ] [MacPherran, Mary "Skeeter"] bound_address {inet[/0:0:0:0:0:0:0:0:9300]}, publish_address {inet[/192.168.1.160:9300]}
[2014-04-23 02:59:18,849][INFO ][cluster.service          ] [MacPherran, Mary "Skeeter"] new_master [MacPherran, Mary "Skeeter"][QHkEcPcJSWevsqSBow7VUQ][inet[/192.168.1.160:9300]], reason: zen-disco-join (elected_as_master)
[2014-04-23 02:59:18,890][INFO ][discovery                ] [MacPherran, Mary "Skeeter"] elasticsearch/QHkEcPcJSWevsqSBow7VUQ
[2014-04-23 02:59:18,938][INFO ][http                     ] [MacPherran, Mary "Skeeter"] bound_address {inet[/0:0:0:0:0:0:0:0:9200]}, publish_address {inet[/192.168.1.160:9200]}
[2014-04-23 02:59:18,938][INFO ][node                     ] [MacPherran, Mary "Skeeter"] {0.90.0}[2323]: started
[2014-04-23 02:59:22,746][INFO ][gateway                  ] [MacPherran, Mary "Skeeter"] recovered [7] indices into cluster_state
[2014-04-23 03:06:08,927][INFO ][cluster.metadata         ] [MacPherran, Mary "Skeeter"] [cities] deleting index
[2014-04-23 03:06:09,088][INFO ][cluster.metadata         ] [MacPherran, Mary "Skeeter"] [cities] creating index, cause [api], shards [5]/[1], mappings [city]
[2014-04-23 03:06:31,152][INFO ][cluster.metadata         ] [MacPherran, Mary "Skeeter"] [cities] update_mapping [city] (dynamic)
[2014-04-23 03:11:44,729][INFO ][cluster.metadata         ] [MacPherran, Mary "Skeeter"] [states] deleting index
[2014-04-23 03:11:44,935][INFO ][cluster.metadata         ] [MacPherran, Mary "Skeeter"] [states] creating index, cause [api], shards [5]/[1], mappings [state]
[2014-04-23 03:12:08,613][INFO ][cluster.metadata         ] [MacPherran, Mary "Skeeter"] [states] update_mapping [state] (dynamic)
[2014-04-23 03:12:26,537][INFO ][cluster.metadata         ] [MacPherran, Mary "Skeeter"] [states] update_mapping [state] (dynamic)
[2014-04-23 04:53:55,896][WARN ][monitor.jvm              ] [MacPherran, Mary "Skeeter"] [gc][ParNew][6327][16] duration [7.7s], collections [1]/[8.5s], total [7.7s]/[8.7s], memory [164mb]->[69.8mb]/[1011.2mb], all_pools {[Code Cache] [4.4mb]->[4.4mb]/[48mb]}{[Par Eden Space] [102.5mb]->[760.7kb]/[102.5mb]}{[Par Survivor Space] [12.7mb]->[10.8mb]/[12.7mb]}{[CMS Old Gen] [48.7mb]->[58.2mb]/[896mb]}{[CMS Perm Gen] [23mb]->[23mb]/[128mb]}
[2014-04-23 05:22:45,385][INFO ][node                     ] [Kamuu] {0.90.0}[2354]: initializing ...
[2014-04-23 05:22:45,446][INFO ][plugins                  ] [Kamuu] loaded [analysis-kuromoji], sites []
[2014-04-23 05:22:51,825][INFO ][node                     ] [Kamuu] {0.90.0}[2354]: initialized
[2014-04-23 05:22:51,825][INFO ][node                     ] [Kamuu] {0.90.0}[2354]: starting ...
[2014-04-23 05:22:52,126][INFO ][transport                ] [Kamuu] bound_address {inet[/0:0:0:0:0:0:0:0:9300]}, publish_address {inet[/192.168.1.160:9300]}
[2014-04-23 05:22:55,529][INFO ][cluster.service          ] [Kamuu] new_master [Kamuu][tqYceQrkQuSLiyrfsVBkww][inet[/192.168.1.160:9300]], reason: zen-disco-join (elected_as_master)
[2014-04-23 05:22:55,567][INFO ][discovery                ] [Kamuu] elasticsearch/tqYceQrkQuSLiyrfsVBkww
[2014-04-23 05:22:55,618][INFO ][http                     ] [Kamuu] bound_address {inet[/0:0:0:0:0:0:0:0:9200]}, publish_address {inet[/192.168.1.160:9200]}
[2014-04-23 05:22:55,618][INFO ][node                     ] [Kamuu] {0.90.0}[2354]: started
[2014-04-23 05:23:00,853][INFO ][gateway                  ] [Kamuu] recovered [7] indices into cluster_state
[2014-04-23 20:53:19,737][INFO ][monitor.jvm              ] [Kamuu] [gc][ParNew][55767][11] duration [704ms], collections [1]/[1.6s], total [704ms]/[1.2s], memory [150.2mb]->[50.3mb]/[1011.2mb], all_pools {[Code Cache] [2.3mb]->[2.3mb]/[48mb]}{[Par Eden Space] [102.5mb]->[1.7mb]/[102.5mb]}{[Par Survivor Space] [652.2kb]->[1.4mb]/[12.7mb]}{[CMS Old Gen] [47.1mb]->[47.1mb]/[896mb]}{[CMS Perm Gen] [22.8mb]->[22.8mb]/[128mb]}
